spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BraveAutoConfiguration.java

```diff
@@ -244,6 +245,19 @@ BaggagePropagationCustomizer remoteFieldsBaggagePropagationCustomizer() {
 			};
 		}
 
+		@Bean
+		@Order(0)
+		BaggagePropagationCustomizer localFieldsBaggagePropagationCustomizer() {
+			return (builder) -> {
+				final List<String> localFields = new ArrayList<>(
```

Comment:
I don't think it's a good idea to use the `getFields()` for both remote and local fields. In https://github.com/spring-projects/spring-boot/issues/33991 we want to add tagged fields. I'd prefer an explicit configuration property for that, as we have it for the remote fields.

Reply:
I have missed that conversation for some reason, but I have implemented both local and tagged fields. I also think it is to have the extra property fields.